dotnet new project launch.json and tasks.json invalid UnoQuickStart as project names

### Current behavior

Linux host, VS Code environment, prerequisites in place, dotnet build and run works.

After a `dotnet new unoapp-winui` or dotnet new `unoapp-winui-net6` the generated launch.json and tasks.json contains invalid project names, e.g. (relevant parts listed):

(launch.json)
```
      "webRoot": "${workspaceFolder}/UnoQuickStart.Wasm",

      "program": "${workspaceFolder}/UnoQuickStart.Skia.Gtk/bin/Debug/net5.0/UnoQuickStart.Skia.Gtk.dll",

```
(tasks.json)
```
        "build",
        "${workspaceFolder}/UnoQuickStart.Wasm/UnoQuickStart.Wasm.csproj",

        "build",
        "${workspaceFolder}/UnoQuickStart.Skia.Gtk/UnoQuickStart.Skia.Gtk.csproj",

```
Making the launch of the Debug session impossible.

### Expected behavior

UnoQuickStart project names replaced with the given project name

### How to reproduce it (as minimally and precisely as possible)

1. Install the dotnet new templates (version 4.0.11)
2. dotnet new unoapp-winui -o DemoApplication -ios=false -android=false -macos=false -wd=false -sw=false --vscode
3. Open folder in Code
4. Select Skia.GTK as debug project
4. Hit F5
5. Click Abort in the dialog
6. Observe the text in Terminal window: 

```
MSBUILD : error MSB1009: Project file does not exist.
Switch: /home/user/sources/DemoApplication/UnoQuickStart.Skia.Gtk/UnoQuickStart.Skia.Gtk.csproj
```

### Workaround

1. Press Shift-Ctrl-H
2. In the Search field type UnoQuickStart.
3. In the Replace field type DemoApplication.  ((or the generated project name))
4. Press Replace All
5. Hit F5
6. The project will start in Debug session (Hello World appears)

### Works on UWP/WinUI

_No response_

### Environment

Other

### NuGet package version(s)

$ dotnet new -u

Currently installed items:
   Uno.ProjectTemplates.Dotnet
      Version: 4.0.11
      Details:
         Author: nventive
         NuGetSource: https://api.nuget.org/v3/index.json
      Templates:
         Cross-Platform UI Tests Library (unoapp-uitest) C#
         Uno Platform WebAssembly Head for Xamarin.Forms (wasmxfhead) C#
         Multi-Platform App (unoapp) C#
         Multi-Platform App (net6) (unoapp-net6) C#
         Cross-Platform Library (unolib) C#
         Cross-Platform App (Prism) (unoapp-prism) C#
         Multi-Platform App (WinUI) (unoapp-winui) C#
         Multi-Platform App net6 (WinUI) (unoapp-winui-net6) C#
         Cross-Runtime Library (unolib-crossruntime) C#
      Uninstall Command:
         dotnet new --uninstall Uno.ProjectTemplates.Dotnet


### Affected platforms

Solution Templates

### IDE

Visual Studio Code

### IDE version

1.62.3 but it doesn't matter

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

--- a/src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp-winui-net6/.vscode/launch.json
+++ b/src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp-winui-net6/.vscode/launch.json
@@ -12,13 +12,13 @@
       "type": "pwa-chrome",
       "request": "launch",
       "url": "http://localhost:5000",
-      "webRoot": "${workspaceFolder}/UnoQuickStart.Wasm",
+      "webRoot": "${workspaceFolder}/UnoWinUIQuickStart.Wasm",
       "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
       "server": {
         "runtimeExecutable": "dotnet",
         "program": "run",
         "outputCapture": "std",
-        "cwd": "${workspaceFolder}/UnoQuickStart.Wasm"
+        "cwd": "${workspaceFolder}/UnoWinUIQuickStart.Wasm"
       }
     },
     {
@@ -30,12 +30,12 @@
       "request": "launch",
       "preLaunchTask": "build-skia-gtk",
       // If you have changed target frameworks, make sure to update the program path.
-      "program": "${workspaceFolder}/UnoQuickStart.Skia.Gtk/bin/Debug/net5.0/UnoQuickStart.Skia.Gtk.dll",
+      "program": "${workspaceFolder}/UnoWinUIQuickStart.Skia.Gtk/bin/Debug/net5.0/UnoWinUIQuickStart.Skia.Gtk.dll",
       "args": [],
       "env": {
         "DOTNET_MODIFIABLE_ASSEMBLIES": "debug"
       },
-      "cwd": "${workspaceFolder}/UnoQuickStart.Skia.Gtk",
+      "cwd": "${workspaceFolder}/UnoWinUIQuickStart.Skia.Gtk",
       // For more information about the 'console' field, see https://aka.ms/VSCode-CS-LaunchJson-Console
       "console": "internalConsole",
       "stopAtEntry": false

--- a/src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp-winui-net6/.vscode/tasks.json
+++ b/src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp-winui-net6/.vscode/tasks.json
@@ -7,7 +7,7 @@
       "type": "process",
       "args": [
         "build",
-        "${workspaceFolder}/UnoQuickStart.Wasm/UnoQuickStart.Wasm.csproj",
+        "${workspaceFolder}/UnoWinUIQuickStart.Wasm/UnoWinUIQuickStart.Wasm.csproj",
         "/property:GenerateFullPaths=true",
         "/consoleloggerparameters:NoSummary"
       ],
@@ -19,7 +19,7 @@
       "type": "process",
       "args": [
         "publish",
-        "${workspaceFolder}/UnoQuickStart.Wasm/UnoQuickStart.Wasm.csproj",
+        "${workspaceFolder}/UnoWinUIQuickStart.Wasm/UnoWinUIQuickStart.Wasm.csproj",
         "/property:GenerateFullPaths=true",
         "/consoleloggerparameters:NoSummary"
       ],
@@ -31,7 +31,7 @@
       "type": "process",
       "args": [
         "build",
-        "${workspaceFolder}/UnoQuickStart.Skia.Gtk/UnoQuickStart.Skia.Gtk.csproj",
+        "${workspaceFolder}/UnoWinUIQuickStart.Skia.Gtk/UnoWinUIQuickStart.Skia.Gtk.csproj",
         "/property:GenerateFullPaths=true",
         "/consoleloggerparameters:NoSummary"
       ],
@@ -43,7 +43,7 @@
       "type": "process",
       "args": [
         "publish",
-        "${workspaceFolder}/UnoQuickStart.Skia.Gtk/UnoQuickStart.Skia.Gtk.csproj",
+        "${workspaceFolder}/UnoWinUIQuickStart.Skia.Gtk/UnoWinUIQuickStart.Skia.Gtk.csproj",
         "/property:GenerateFullPaths=true",
         "/consoleloggerparameters:NoSummary"
       ],

--- a/src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp-winui/.vscode/launch.json
+++ b/src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp-winui/.vscode/launch.json
@@ -12,13 +12,13 @@
       "type": "pwa-chrome",
       "request": "launch",
       "url": "http://localhost:5000",
-      "webRoot": "${workspaceFolder}/UnoQuickStart.Wasm",
+      "webRoot": "${workspaceFolder}/UnoWinUIQuickStart.Wasm",
       "inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",
       "server": {
         "runtimeExecutable": "dotnet",
         "program": "run",
         "outputCapture": "std",
-        "cwd": "${workspaceFolder}/UnoQuickStart.Wasm"
+        "cwd": "${workspaceFolder}/UnoWinUIQuickStart.Wasm"
       }
     },
     {
@@ -30,12 +30,12 @@
       "request": "launch",
       "preLaunchTask": "build-skia-gtk",
       // If you have changed target frameworks, make sure to update the program path.
-      "program": "${workspaceFolder}/UnoQuickStart.Skia.Gtk/bin/Debug/net5.0/UnoQuickStart.Skia.Gtk.dll",
+      "program": "${workspaceFolder}/UnoWinUIQuickStart.Skia.Gtk/bin/Debug/net5.0/UnoWinUIQuickStart.Skia.Gtk.dll",
       "args": [],
       "env": {
         "DOTNET_MODIFIABLE_ASSEMBLIES": "debug"
       },
-      "cwd": "${workspaceFolder}/UnoQuickStart.Skia.Gtk",
+      "cwd": "${workspaceFolder}/UnoWinUIQuickStart.Skia.Gtk",
       // For more information about the 'console' field, see https://aka.ms/VSCode-CS-LaunchJson-Console
       "console": "internalConsole",
       "stopAtEntry": false

--- a/src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp-winui/.vscode/tasks.json
+++ b/src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp-winui/.vscode/tasks.json
@@ -7,7 +7,7 @@
       "type": "process",
       "args": [
         "build",
-        "${workspaceFolder}/UnoQuickStart.Wasm/UnoQuickStart.Wasm.csproj",
+        "${workspaceFolder}/UnoWinUIQuickStart.Wasm/UnoWinUIQuickStart.Wasm.csproj",
         "/property:GenerateFullPaths=true",
         "/consoleloggerparameters:NoSummary"
       ],
@@ -19,7 +19,7 @@
       "type": "process",
       "args": [
         "publish",
-        "${workspaceFolder}/UnoQuickStart.Wasm/UnoQuickStart.Wasm.csproj",
+        "${workspaceFolder}/UnoWinUIQuickStart.Wasm/UnoWinUIQuickStart.Wasm.csproj",
         "/property:GenerateFullPaths=true",
         "/consoleloggerparameters:NoSummary"
       ],
@@ -31,7 +31,7 @@
       "type": "process",
       "args": [
         "build",
-        "${workspaceFolder}/UnoQuickStart.Skia.Gtk/UnoQuickStart.Skia.Gtk.csproj",
+        "${workspaceFolder}/UnoWinUIQuickStart.Skia.Gtk/UnoWinUIQuickStart.Skia.Gtk.csproj",
         "/property:GenerateFullPaths=true",
         "/consoleloggerparameters:NoSummary"
       ],
@@ -43,7 +43,7 @@
       "type": "process",
       "args": [
         "publish",
-        "${workspaceFolder}/UnoQuickStart.Skia.Gtk/UnoQuickStart.Skia.Gtk.csproj",
+        "${workspaceFolder}/UnoWinUIQuickStart.Skia.Gtk/UnoWinUIQuickStart.Skia.Gtk.csproj",
         "/property:GenerateFullPaths=true",
         "/consoleloggerparameters:NoSummary"
       ],

